templates/types/streaming/fastapi/app/api/routers/chat.py

```diff
@@ -168,3 +175,14 @@ async def chat_request(
         result=_Message(role=MessageRole.ASSISTANT, content=response.response),
         nodes=_SourceNodes.from_source_nodes(response.source_nodes),
     )
+
+
+@r.get("/config")
+async def chat_config() -> _ChatConfig:
+    conversation_starters = os.getenv("CONVERSATION_STARTERS")
+    starter_questions = (
+        conversation_starters.strip().split("\n") if conversation_starters else None
```

Comment:
@thucpn how about we do not convert the starter_questions if the `CONVERSATION_STARTERS` is an empty string? It's normal that the user set it as empty string instead of letting it as null